CMakeLists.txt

```
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.0.2)
```

```suggestion
cmake_minimum_required(VERSION 3.16)
```

This is recommended by my CMake. If I don't specify `cmake_minimum_required`, it tells me to specify 3.16.

---

Yes, because you have 3.16 installed. I ran through the docs to check before I submitted the PR, all of the commands I've used are compatible with 3.0.2. Setting a higher requirement just makes things more difficult for people on systems running older distros for no benefit. e.g. Ubuntu 18.04 LTS, supported until 2023, ships with 3.10.2

---

Makes sense.

---

By the way, why exactly 3.0.2? Why not 3.0.0?